ProteoformSuiteGUI/ExperimentExperimentComparison.cs

```diff
@@ -20,8 +20,8 @@ public ExperimentExperimentComparison()
         {
             InitializeComponent();
             this.dgv_EE_Peaks.MouseClick += new MouseEventHandler(dgv_EE_Peak_List_CellClick);
-            this.ct_EE_Histogram.MouseMove += new MouseEventHandler(ct_EE_Histogram_MouseMove);
```

Comment:
Oh, I see now how you changed this. Is it a solution for the flickering tooltip?